### PR TITLE
EOS-19514 scripts: Verify lnet in config and prepare phases

### PR DESCRIPTION
**Pull request by atulsdeshmukh2312**
Signed-off-by: Atul Deshmukh <atul.deshmukh@seagate.com>

### CODE REVIEW

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py [hunk 1]
```diff
@@ -129,8 +138,28 @@ def validate_motr_rpm(self):
     sys.stdout.write(f"Checking for {MOTR_SYS_FILE}\n")
     validate_file(MOTR_SYS_FILE)
 
+def verify_lnet(self):
+    sys.stdout.write(f"Doing ping to nids.\n")
```
Comment (huanghua78) [inline scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:142]: Please fix these warnings.

**Comment by atulsdeshmukh2312** [comment]
Successfully run on jenkins
http://ssc-vm-2590.colo.seagate.com:8080/job/mini-provision-threenode/94/console

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py [hunk 2]
```diff
@@ -129,8 +138,28 @@ def validate_motr_rpm(self):
...
+        if not check_services(self, ["lnet"]):
+            sys.stdout.write("lnet is not up. Restaring lnet.\n")
+            restart_services(self, ["lnet"])
+            sys.stdout.write("Doing ping to nids after 5 seconds.\n")
+            execute_command_without_exception(self, "sleep 5")
+            ret = lnet_self_ping(self)
+        else:
+            sys.stdout.write("lnet is up. Doing ping to nids after 5 seconds.\n")
+            execute_command_without_exception(self, "sleep 5")
+            ret = lnet_self_ping(self)
```
Comment (huanghua78) [inline scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:156]: These two commands
```
            execute_command_without_exception(self, "sleep 5")
            ret = lnet_self_ping(self)
```
are duplicated. 
They will be called in both cases.
So, we can call it outside of this "if ... else ...".

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py [hunk 3]
```diff
@@ -179,6 +208,9 @@ def configure_lnet(self):
         fp.write(lnet_config)
 
     restart_services(self, ["lnet"])
+    # Ping to nid
+    sys.stdout.write("Doing ping to nids\n")
+    lnet_self_ping(self)
```
Comment (huanghua78) [inline scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:213]: Do we need to return the rc of lnet_self_ping() to caller?

**Review by huanghua78 — APPROVED** [review]

**Comment by atulsdeshmukh2312** [comment]
Can we run VM deployment once again.

Since in VM deployment, motr phases are passed. Other componant is failing.
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/Motr/346/artifact/setup.log

**Comment by yeshpal-jain-seagate** [comment]
please test this

**Comment by gowthamchinna** [comment]
run vm-deployment

**Comment by gowthamchinna** [comment]
test this please

**Comment by gowthamchinna** [comment]
test this please

**Comment by atulsdeshmukh2312** [comment]
mini provisioner deployment is passed as below. But result is not updated in mini-provisioner check.
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Mini-Provisioning/job/cortx-motr/582/

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py (hunk 3 above)
Comment (atulsdeshmukh2312) [reply scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py]: Done 19c9afc

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py (hunk 2 above)
Comment (atulsdeshmukh2312) [reply scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py]: Done 19c9afc

**Comment by huanghua78** [comment]
@atulsdeshmukh2312 is this patch already tested?

**Comment by atulsdeshmukh2312** [comment]
VM deployment is passed
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/Motr/517/

**Comment by atulsdeshmukh2312** [comment]
test this please

**Comment by atulsdeshmukh2312** [comment]
New PR on tmain branch
https://github.com/Seagate/cortx-motr/pull/675

**Comment by atulsdeshmukh2312** [comment]
New PR is PR#720
https://github.com/Seagate/cortx-motr/pull/720
Closing this PR#577.

**Pull request closed without merging**